sympy/utilities/lambdify.py

```diff
@@ -336,6 +336,22 @@ def lambdify(args: Iterable, expr, modules=None, printer=None, use_imps=True,
         (if ``args`` is not a string) - for example, to ensure that the
         arguments do not redefine any built-in names.
 
+    cse : bool or callable, optional
+        When vectorializing a large expression as efficient as possible,
+        set this parameter to True.
+        Whether or not common subexpression elimination should be performed on
+        the expressions, This can potentially improve performance of the generated
+
+        However, the step of finding redundant paterns can be slow.
+        So, if the creation of the 'lambdify' function should be fast,
+        set this parameter to False.
+
+        Whether or not common subexpression elimination should be performed on
+        the expressions. This can potentially improve performance of the generated
+        function. When ``True``, then ``sympy.simplify.cse`` is used, the user
+        may pass their own function matching the signature of said function.
```

Comment:
Instead of using "None" to signify this low memory option, I think it would be better to show an example of using the callable option, in which you use the low memory cse function.

Reply:
Do you think it's ok to do both or do you want the `cse` option to be strictly binary? In either case something like the following could be added:
```python
from sympy.simplify.cse_main import cse_minimize_memory
f = lambdify(exprs, cse=lambda x: cse(x, postprocess=cse_minimize_memory)
```
```

Reply:
It just seems odd to link "None" to this specific cse implementation. Having cse is a super common case, but customizing cse optimizations isn't so common and there are many options. Your example seems like a universal way to handle it.